kickstarter/android-oss
# Allow backers to add a UnionPay card in Account Settings

# What ❓
- Added `Union pay` to list of allowed cards and added vector drawable for union pay. Also updated `NewCardFragmentViewModelTest`. 

# How to QA? 🤔
- On `Staging` add a `Union Pay` card and make sure it successfully adds the card. Then go to back a (non-US) project and make sure that you see the following message in the Webview.
>This credit card is not accepted in Mexico.

# Story 📖
[Allow backers to add a UnionPay card in Account Settings](https://trello.com/c/RSHO2bpf/1267-allow-backers-to-add-a-unionpay-card-in-account-settingsandroid)

# See 👀
![Mar-11-2019 15-58-24](https://user-images.githubusercontent.com/16387538/54153672-8b62fb80-4416-11e9-9199-f988f23d6932.gif)
<img src=https://user-images.githubusercontent.com/16387538/54153832-e137a380-4416-11e9-83cc-af2a788cb16d.jpg width=230 />
